app/templates/dayview.html

```diff
@@ -10,38 +10,49 @@
+                      <a href="https://www.horoscope.com/zodiac-signs/{{zodiac.name}}" title="Zodiac-sign: {{zodiac.name}}" target="_blank"><img src="{{ url_for('static', path='/images/zodiac/' + zodiac.name + '.svg') }}" alt="zodiac sign" width="14em" height="13em"></a>
+                  </div>
+                {% endif %}
+            {% else %}
+                <span class="fw-bold text-white date-nums">{{day}} / {{month}}</span>
             {% endif %}
         </div>
         <div class="schedule">
             <div class="container times bg-primeary position">
-                {% for i in range(24)%}
-                    <div class="row bg-transparent hourmark">
-                    {% set i = i|string() %}
-                    {{i.zfill(2)}}:00
-                    </div>
-                {% endfor %}
+                {% if view == 'day'%}
+                    {% for i in range(24)%}
+                        <div class="row bg-transparent hourmark">
+                        {% set i = i|string() %}
+                        {{i.zfill(2)}}:00
+                        </div>
+                    {% endfor %}
+                {% endif %}
             </div>
             <div class="eventgrid">
                 {% for event, attr in events %}
                     {% set totaltime = 'visible'%}
-                    {% if attr.length < 60 %}
-                            {% set size = 'title_size_small' %}
-                            {% set totaltime = 'invisible'%}
-                        {% if attr.length < 45 %}
-                                {% set size = 'title_size_Xsmall' %}
-                            {% if attr.length < 30 %}
-                                    {% set size = 'title_size_tiny' %}
+                    {% if attr.length <= 90 %}
+                        {% set size = 'title_size_small' %}
+                        {% if attr.length <= 60 %}
+                                {% set totaltime = 'invisible'%}
+                            {% if attr.length < 45 %}
+                                    {% set size = 'title_size_Xsmall' %}
+                                {% if attr.length < 30 %}
+                                        {% set size = 'title_size_tiny' %}
+                                {% endif %}
                             {% endif %}
                         {% endif %}
                     {% endif %}
-                    <div id="event{{event.id}}" class="text-truncate px-5 d-flex flex-column justify-content-evenly event" style="background-color: {{attr.color}}; grid-row: {{attr.grid_position}}; max-hight:1.5rem;">
+                    {% if view == 'week'%}
+                        {% set size = 'week-view-title' %}
+                    {% endif %}
+                    <div id="event{{event.id}}" class="text-truncate px-2 d-flex flex-column justify-content-evenly event" style="background-color: {{attr.color}}; grid-row: {{attr.grid_position}}; max-hight:1.5rem;">
```

Comment:
Set the style in the CSS according to the class, if possible.

Reply:
it will be very hard to save css class for each position or color becouse there gonna be so many options. there are 96 rows on this grid and every event has a diffrent position(96**2 options?). same for color as we intend to allow all the hex options eventualy...

Reply:
Walla sababa